### ISSUE TITLE
PR Review: Allow adding review comments on unchanged lines in a PR diff

### ISSUE THREAD

**Issue by anthonykim1** [microsoft/vscode#305719]
## Problem
When reviewing a PR inside VS Code, the `+` button to "start discussion" or add a comment is only available on lines that were changed by the PR. There is no obvious way to add a comment on a line of code in the same file that was **not** changed by the PR.

## Expected Behavior
Users should be able to add review comments on any line in a file that is part of a PR diff, not just on the changed lines. This is especially important for screen reader users who may want to comment on context surrounding a change.

## Context
Reported during accessibility meeting on March 20th. A screen reader user asked if there was a way to start a discussion on unchanged lines in the PR diff view.

/cc @meganrogge @alexr00
